src/vos/ilog.h

```diff
@@ -261,6 +261,15 @@ ilog_fetch_finish(struct ilog_entries *entries);
 	for (entry = &(ents)->ie_entries[(ents)->ie_num_entries - 1];	\
 	     entry != &(ents)->ie_entries[-1]; entry--)
 
+/** Fetch the address of the timestamp index from the ilog entry
```

Comment:
Instead of "ilog entry", it should read "ilog"?

Reply:
Indeed, can fix.  Can I do follow up to fix small things?

Reply:
> Indeed, can fix. Can I do follow up to fix small things?

Yes, of course.